js/runtime.ts

```diff
@@ -131,8 +131,8 @@ export function makeDefine(fileName: string): AmdDefine {
         return localExports;
       } else if (dep === "typescript") {
         return ts;
-      } else if (dep === "deno") {
-        return deno;
+      // } else if (dep === "deno") {
+      //   return deno;
```

Comment:
It isn't material to this PR, but during this PR I uncovered what is an error at the moment.  Currently the import of `deno.ts` is commented out:

https://github.com/ry/deno/blob/b44c235579ac9fbef14d4d33e21f266d0a2ff73e/js/runtime.ts#L17

But TypeScript didn't flag this as an error, because of the way that `deno.d.ts` is declared, in that it appears to be a "global" variable named `deno` (which it isn't).  So actually returning `deno` here would returned `undefined`.  When I was doing the work for `lib.deno.d.ts` I too mistook `deno` for a global variable and then realised it wasn't.

For cleaness I can leave what is "broken" broken for now and it can be addressed when `deno.ts` is fixed/updated/restored.

Reply:
> For cleaness I can leave what is "broken" broken for now and it can be addressed when deno.ts is fixed/updated/restored.

Yeah that works. Thank you.